src/components/MainArea/MovieList/Movie/MovieCard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled, { css } from "styled-components";
+import PropTypes from 'prop-types'
+import InfoSection from './InfoSection';
+
+const MovieCardStyled = styled.div`
+    width: min-content;
+    height: min-content;
+    margin: 15px 15px 15px 15px;    
+    overflow: hidden;
+`;
+
+function MovieCard(props) {
+    const {imgUrl, name, ...otherProps} = props;
+    return(
+        <>
+            <MovieCardStyled>
+                <img src={imgUrl} alt={name} />
+                <InfoSection genres = {props.genres} productionYear= {props.productionYear} name = {props.name}></InfoSection>
```

Comment:
No need for space here genres = {props.genres} and in the ongoing ones

Reply:
fixed